app/models/wizard/steps/base.rb

```diff
@@ -9,12 +9,26 @@ class Base
       def initialize(session, attributes = {})
         @session = session
 
+        clean_session if attributes.empty?
```

Comment:
Ensure we only clean the session while on show action. attributes will be non-empty only on #create actions.